docs/source/utils/notebook.py

```diff
@@ -1,16 +1,38 @@
+import re
+from typing import Dict, List, Optional
+
 from jupytext import jupytext
 
+start_pattern = re.compile(r'# %% \[markdown\]\n"""\n# (\d\. .*)\n\n[\S\s]*?"""\n')
+
+
+def add_installation_cell_into_example(default: str, dependencies: Optional[Dict[str, List[str]]] = None):
```

Comment:
I think, that previous function name `add_installation_cell_into_py`was better

Reply:
may be `insert_installation_cell_into_notebook` ?

Reply:
I think that `example` is actually better than both of the suggested names because the example file is not just a python or py:percent formatted file - it's a file of the specific structure, with name, number, header, etc.

Reply:
When we use it, we will use it only for `.py`-files in `conf.py` also we have special `pattern` that is applicable only for   `py:percent` because of that i do not get your point and suggest `insert_installation_cell_into_py_example` name